src/content/resources/cpp/multidimensional-arrays.md

```diff
@@ -0,0 +1,122 @@
+---
+authors:
+  - "alearori#8661"
+created_at: 2019/11/15
+title: "Multidimensional Arrays"
+---
+
+## What is a multidimensional array?
+
+To put simply, a multidimensional array is an array that contains another array of another array... of another array of objects. It is commonly used for making matricies, but it can also be used for storing information such as if you use `string`. 
```

Comment:
Could probably word it as "such as strings", "such as if you use x" sounds a little weird, not a big deal though.

Reply:
Okay, I changed the wording to match that.